### ISSUE TITLE
Issue displaying errors from backend

### ISSUE THREAD

**Issue by dyaffe** [estuary/ui#557]
**Bug**
[LogRocket session](https://app.logrocket.com/tue20y/estuarytest/s/5-08f44327-caa0-4a65-bdb3-807f34f3c87d/0?t=1680850870195.6084) shows a user encountering a connector error

When they expanded the shard details on the capture page, the stderr snippet under "Shard Replica Processing Errors" appears to show the start of the stderr snippet rather than the final bits where we actually emitted an error message.
According to the DOM inspector the error message was present, it just doesn't appear to be visible onscreen and the stderr box can't be scrolled to see the final lines.

This could be related to the large, multi-line log message (beginning {"alterOptions"...) which also appears to be reflowed incorrectly.  It is possible that the user saw the message just fine and the problem is LogRocket displaying things wrong. I'm not very familiar with how LogRocket does its thing and whether that's a possibility.

**Comment by travjenkins** [comment]
There are the scrollbars showing up for the section here:
![image](https://user-images.githubusercontent.com/270078/231188138-4f28f13e-bc1c-45e0-9abe-ca53a4f37bc1.png)
![image](https://user-images.githubusercontent.com/270078/231188177-4ce1c943-7254-42c8-8787-e8a976ba357b.png)


Tested in production with errors that are pretty long and they are fine:
![image](https://user-images.githubusercontent.com/270078/231172656-048bb48e-7dc1-46c6-a293-2abf66177fd1.png)

I checked LogRocket when I tested and saw myself scrolling. So I do not think the user scrolled here.

As for the reflowing... I do not see anything special the UI is doing with that. It just takes the string given to it and and displays it. The strings coming back do contain new lines in them so the wrapping should be controlled by that.
